internal/handlers/pg/pjson/schema.go

```diff
@@ -62,6 +62,40 @@ const (
 	elemTypeLong      elemType = "long"
 )
 
+// GetTypeOfValue returns pjson type of supported value.
+func GetTypeOfValue(v any) string {
```

Comment:
That's one bit I'm concerned about, but it is not used in this PR, so we could move to that other PR

Reply:
Forgot about this one.